ecommerce/serializers.py

```diff
@@ -367,15 +367,26 @@ def _update_items(cls, basket, items):
             item = items[0]
             product_id = item.get("product_id")
             run_ids = item.get("run_ids")
-            product = models.Product.objects.get(id=product_id)
+            try:
+                # product_id = product_id
```

Comment:
unnecessary comment

Reply:
and I think, better to check the `None` state before utliziing `product_id` below. 
```
if product_id: 
```

Reply:
it's not needed because it's already been validated in `validate_items`. Here is the reference code https://github.com/mitodl/mitxpro/blob/master/ecommerce/serializers.py#L520`.

If you still ask to add, I will add it.

Reply:
It does all kinds of validations either a `product_id` or product with `product_id` exist etc.